astroplan/utils.py

```diff
@@ -3,22 +3,19 @@
                         unicode_literals)
 
 # Standard library
-import urllib.error
 import warnings
 
 # Third-party
 import numpy as np
-from astropy.utils.data import download_file, clear_download_cache
-from astropy.utils import iers
+from astropy.utils.iers import IERS_Auto
 from astropy.time import Time
 import astropy.units as u
-from astropy.utils.data import _get_download_cache_locs, CacheMissingWarning
 from astropy.coordinates import EarthLocation
 
 # Package
 from .exceptions import OldEarthOrientationDataWarning
 
-__all__ = ["get_IERS_A_or_workaround", "download_IERS_A",
```

Comment:
This has been an exposed public API, removing without a deprecation period is not a good practice. Ideally code review is in place to pick up things like this